lib/tasks/import.rake

```diff
@@ -0,0 +1,78 @@
+namespace :host_reports do
+  def config_report_help(id, host_id, reported_at, status, metrics, type, origin)
+    config_report = Hash["id": id, "host_id": host_id, "reported_at": reported_at,
+      "metrics": metrics, "type": type, "origin": origin]
+    ConfigReport.create(config_report)
```

Comment:
Use `create!` to see exception, many RoR methods have bang version which does the very same but it raises an exception on error instead of returning bool.

Some ideas on why this might not work: keys as strings instead symbols, use of type (do not set type as it will be set automatically by ActiveRecord via STI stack)